src/App.css

```diff
@@ -59,11 +59,14 @@
 }
 
 .paragraph {
+  display: flex;
+  align-items: center;
   font-size: 2.6vh;
 }
 
 .paragraph a{
-    color: rgba(0,0,0,.45);
+  color: rgba(0,0,0,.45);
+  margin-right: 10px;
```

Comment:
@umershaikh97 10px margin not looking good in mobile view... change margin-right to `margin-right: 1vh;`

Reply:
Sure

Reply:
@NomanGul Done!